CONTRIBUTING.md

```diff
@@ -58,11 +59,19 @@ If you are still having issues or if your command prompt is throwing an error th
 Check to see where the Python you're running is located (`which python`). You should see something like `/Users/your_account_shortname/.pyenv/shims/python`.
 If you don't see something similar, you may have several versions of Python installed elsewhere (via Anaconda or Homebrew). If (and only if) you'd like to clear out any previous homebrew-based installs, type `brew uninstall --ignore-dependencies python3 && brew uninstall --ignore-dependencies python`.
 
-### Database commands.
+### Database migrations.
 
-- Create and Seed the Database: `pipenv run python manage.py create`
-- Delete, create, Seed the Database: `pipenv run python manage.py recreate`
-- Delete the Database: `pipenv run python manage.py drop`
+Database migrations are managed through [Alembic](https://alembic.sqlalchemy.org/en/latest/index.html). After making a change to a model, a database migration is necessary. 
```

Comment:
@yunnie I would like to get this merged, but since we are not going to be using this right away. Should we add a note here, something like "Migrations are currently not being used, Please ignore for now" or simply remove the doc update and put it back in later? What do you think?

Reply:
Sorry, I forgot to get back to you on this. I think it makes most sense to edit the docs back to back to the original instruction for creating, deleting and seeding the database. And to add a note stating that migrations are currently not being used "please ignore for now". I'll make the changes to the docs, if you haven't already.

Reply:
I just added a line in the CONTRIBUTING.md that instructs users not to use database migrations for development.